pkg/operator/ceph/controller/cluster_info.go

```diff
@@ -50,8 +51,13 @@ const (
 	MonSecretNameKey  = "mon-secret"
 	// AdminSecretName is the name of the admin secret
 	adminSecretNameKey = "admin-secret"
-	CephUsernameKey    = "ceph-username"
-	CephUserSecretKey  = "ceph-secret"
+	// CephUsernameKey for the new operator user
+	CephUsernameKey = "ceph-operator-username"
```

Comment:
And add similar naming for the related consts and vars
```suggestion
	CephOperatorUsernameKey = "ceph-operator-username"
```